apps/src/templates/teacherDashboard/StatsTable.jsx

```diff
@@ -0,0 +1,149 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import {Table, sort} from 'reactabular';
+import wrappedSortable from '../tables/wrapped_sortable';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import orderBy from 'lodash/orderBy';
+
+const styles = {
+  table: {
+    width: '100%'
+  }
+};
+
+class StatsTable extends Component {
+  static propTypes = {
+    section: PropTypes.shape({
+      id: PropTypes.number,
+      students: PropTypes.array
+    }),
+    studentsCompletedLevelCount: PropTypes.object
+  };
+
+  state = {};
+
+  studentsWithCompletedLevelCount = () => {
+    const {section, studentsCompletedLevelCount} = this.props;
+    return section.students.map(student => {
+      return {
+        ...student,
+        completed_levels_count: studentsCompletedLevelCount[student.id] || 0
+      };
+    });
```

Comment:
This could be an expression:
```jsx
return section.students.map( student => ({
  ...student,
  completed_levels_count: studentsCompletedLevelCount[student.id] || 0
}));
```

Reply:
good point! i merged this branch (<20 seconds before your comment came through), but we are doing a round of cleanup this afternoon and i'll make this update and the reassignment update you mentioned for `StatsTable`